javascript/javascript3/week2/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>JS3-weeK3 Homework</title>
+  </head>
+  <body>
+    <div class="converter">
+      <h1>
+        CURRENCY <br />
```

Comment:
You don't really need `<br />` here, good reading https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br